### PR TITLE
Fix sonar cloud issues

### PR DESCRIPTION
**Pull request by s-weigand**
Basically the same as https://github.com/glotaran/pyglotaran/pull/408, but more easy to compare, since it is against its base branch.

### CODE REVIEW

File: glotaran/builtin/models/doas/test/test_doas_model.py [hunk 1]
```diff
@@ -208,7 +209,8 @@ def test_doas_model(suite):
     assert suite.model.valid(suite.parameter)
 
     dataset = suite.sim_model.dataset["dataset1"].fill(suite.sim_model, suite.wanted_parameter)
-    data = xr.DataArray(suite.axis["time"], coords=[("time", suite.axis["time"])])
+    # TODO: Actually use this value for something or delete it.
+    # data = xr.DataArray(suite.axis["time"], coords=[("time", suite.axis["time"])])
```
Comment (jsnel) [inline glotaran/builtin/models/doas/test/test_doas_model.py:213]: odd indeed, it gets overwritten a few lines down

File: glotaran/model/model.py [hunk 2]
```diff
@@ -40,42 +40,42 @@ def from_dict(cls, model_dict_ref: typing.Dict) -> typing.Type["Model"]:
             if hasattr(model, f"set_{name}"):
 
                 # get the set function
-                set = getattr(model, f"set_{name}")
+                model_set = getattr(model, f"set_{name}")
```
Comment (jsnel) [inline glotaran/model/model.py:43]: I can confirm my linter also complained about it. :)

**Review by jsnel — APPROVED** [review]
Look good. A lot of Exception -> ValueError but some other signification improvement in code quality as well, thanks!

File: docs/source/conf.py [hunk 3]
```diff
@@ -9,22 +9,17 @@
...
-
 # -- Project information -----------------------------------------------------
 
+authors = ("Joern Weissenborn", "Joris Snellenburg", "Ivo van Stokkum")
+
 project = "pyglotaran"
-copyright = "2018, Joern Weissenborn, " "Joris Snellenburg, " "Ivo van Stokkum"
-author = "Joern Weissenborn, " "Joris Snellenburg, " "Ivo van Stokkum"
+title = f"{project} Documentation"
+copyright = ", ".join(("2018", *authors))
```
[the remark this answers is not in the document]
Comment (s-weigand) [reply docs/source/conf.py:21]: I often saw date ranges i.e. `2018-2020`, for project where the meintainer changed, the have multiple copyright ranges, with diffrent people holding the copyright for thaose time periods.

File: glotaran/builtin/models/doas/test/test_doas_model.py (hunk 1 above)
Comment (s-weigand) [reply glotaran/builtin/models/doas/test/test_doas_model.py:213]: Guess this is a refactoring artifact, due to linter not complaining.

**Pull request merged**